tests/lax_numpy_test.py

```diff
@@ -4965,6 +4967,11 @@ def testArangeJit(self):
     expected = jtu.with_jax_dtype_defaults(np.arange)(5)
     self.assertAllClose(ans, expected)
 
+  def testArangeJaxpr(self):
+    jaxpr = jax.make_jaxpr(lambda: jnp.arange(5))()
+    jaxpr_zero = jax.make_jaxpr(lambda: jnp.arange(0, 5))()
+    self.assertEqual(str(jaxpr_zero), str(jaxpr))
```

Comment:
This may end up being an overly strict test, since it depends on the exact string representation. I don't think there's a significantly better way to write it for now.

Reply:
Good point! I changed the test to not rely on the exact string representation.